src/groupByKey.ts

```diff
@@ -1,7 +1,19 @@
+interface Obj {}
```

Comment:
```suggestion
interface ItemGroup {}
```
Давай пропишемо цей інтерфейс. Це має бути масив Item'ів (Item[])